client/changelist/change.go

```diff
@@ -38,8 +38,10 @@ type TufChange struct {
 // TufRootData represents a modification of the keys associated
 // with a role that appears in the root.json
 type TufRootData struct {
-	Keys     data.KeyList `json:"keys"`
-	RoleName string       `json:"role"`
+	ReplaceKeys data.KeyList `json:"keys"`
```

Comment:
We need the `keys` field to exist in the JSON. I found `ReplaceKeys` a slightly confusing name: are they keys to be replaced, or keys we are using as replacements? Figured, as we could create a new action `ActionReplace` or similar and just use the `AddKeys` field (which would map to `keys` in JSON), rather than needing 3 key fields in the struct.